peglib.h

```diff
@@ -2173,7 +2337,8 @@ class ParserGenerator
 
         g["Identifier"] <= seq(g["IdentCont"], g["Spacing"]);
         g["IdentCont"]  <= seq(g["IdentStart"], zom(g["IdentRest"]));
-        g["IdentStart"] <= cls("a-zA-Z_\x80-\xff%");
+        g["IdentStart"] <= cho(seq(npd(cls("\x01-\x7f")), dot()), cls("a-zA-Z_%"));
+
```

Comment:
I changed this rule to allow unicode characters in identifiers while keeping this meta grammar pure ascii (because the UTF8 reader will choke on characters above \x7f).